pkg/virt-controller/watch/migration.go

```diff
@@ -631,6 +633,32 @@ func (c *MigrationController) createTargetPod(migration *virtv1.VirtualMachineIn
 		}
 	}
 
+	matchLevelOnTarget := c.clusterConfig.GetMigrationConfiguration().MatchSELinuxLevelOnMigration
+	if matchLevelOnTarget == nil || *matchLevelOnTarget {
```

Comment:
Is this condition correct? If the tunable for this is omitted, it's an opt-in?
Also, just nitpicking, it may be nice to return early if the tunable is false, avoiding the extra indentation
Would probably mean this chunk needs to get extracted to it's own func

Reply:
Done, PTAL!